rclpy/test/test_node.py

```diff
@@ -838,6 +842,97 @@ def test_node_set_parameters_rejection(self):
         self.assertIsInstance(result[0], SetParametersResult)
         self.assertFalse(result[0].successful)
 
+    def test_node_set_paramters_rejection_list(self):
```

Comment:
"parameter" is misspelled here and in other places.
```suggestion
    def test_node_set_parameters_rejection_list(self):
```